packages/phrases-ui/src/locales/ru.ts

```diff
@@ -45,8 +45,8 @@ const translation = {
     reminder: 'Напоминание',
     not_found: '404 Не найдено',
     agree_with_terms: 'Я прочитал и согласен с ',
-    agree_with_terms_modal: 'Чтобы продолжить, пожалуйста, согласитесь с <link></link>.',
-    terms_of_use: 'Условия использования',
```

Comment:
I think this line needs to be in sentence case, at least that's what it is in English scripts. So why do you change the first capital letter?

Reply:
It's all about the [case](https://www.thoughtco.com/russian-cases-4768614) of the word, it was in the wrong case and looked out of the general sentence.

I will look at examples on other Russian sites to see if I can somehow make a sentence in the nominative case and with a capital letter.

Reply:
Ah, don't worry. I'm just curous since I know little about Russian language. You can resolve this comment if you confirm it is OK.

Reply:
I suggest leaving the capital letter, but I don't see how to leave the `Условия использования` in the Nominative case, so I wrote it in the Instrumental case `Условиями использования`.

The one way to keep in Nominative is to replace `согласен (agree)` to `принимаю (accept)`, but I'm afraid it may cause misses in another parts of application, so it's better to do the lesser evil.